specification/vmware/resource-manager/Microsoft.AVS/preview/2019-08-09-preview/vmware.json

```diff
@@ -0,0 +1,1277 @@
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "tags": [
+    {
+      "name": "PrivateClouds",
+      "description": ""
+    },
+    {
+      "name": "Clusters",
+      "description": ""
+    },
+    {
+      "name": "Operations",
+      "description": ""
+    },
+    {
+      "name": "Quotas",
+      "description": ""
+    }
+  ],
+  "parameters": {
+    "subscriptionId": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Unique identifier for the Azure subscription"
+    },
+    "resourceGroupName": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Name of the resource group within the Azure subscription",
+      "x-ms-parameter-location": "method"
+    },
+    "apiVersion": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Version of Azure VMware Solution API to be used with the client request"
+    },
+    "privateCloudName": {
+      "name": "privateCloudName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Name of the private cloud",
+      "x-ms-parameter-location": "method"
+    },
+    "clusterName": {
+      "name": "clusterName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Name of the cluster in the private cloud",
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.AVS/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all of the available operations",
+        "operationId": "Operations_List",
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request has succeeded",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AVS/locations/{location}/checkQuotaAvailability": {
```

Comment:
>AVS [](start = 57, length = 3)

Have you considered expanding the acronym to Microsoft.AzureVMWareSolution. "AVS" is not recognizable at a glance and you're getting a naming convention linter error right now.

Reply:
"AVS" was recommended by our business team and our LT as it is more recognizable to our customers

Reply:
Sounds good (had to ask).

---
In reply to: [419673350](https://github.com/Azure/azure-rest-api-specs/pull/9307#discussion_r419673350) [](ancestors = 419673350)